receiver/windowsperfcountersreceiver/go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/mitchellh/copystructure v1.2.0 // indirect
 	github.com/mitchellh/mapstructure v1.4.3 // indirect
 	github.com/mitchellh/reflectwalk v1.0.2 // indirect
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/scrapertest v0.46.0
```

Comment:
This will have to be bumped to the most recent release.

Reply:
updated